PowerEditor/installer/nativeLang/galician.xml

```diff
@@ -630,14 +630,15 @@ Translation note:
 					<Item id="2219" name="Palabras clave predeterminadas"/>
 					<Item id="2221" name="Palabras clave definidas polo usuario"/>
 					<Item id="2225" name="Linguaxe:"/>
-					<Item id="2226" name="Habilitar cor global das fontes"/>
-					<Item id="2227" name="Habilitar cor global do fondo"/>
-					<Item id="2228" name="Habilitar tipo global de fonte"/>
-					<Item id="2229" name="Habilitar tamaño global de fonte"/>
-					<Item id="2230" name="Habilitar estilo global en grosa"/>
-					<Item id="2231" name="Habilitar estilo global en cursiva"/>
-					<Item id="2232" name="Habilitar estilo global en suliñado"/>
-					<Item id="2234" name="Ir á configuración"/>
+					<Item id="2226" name="Forzar a cor de primeiro plano para tódolos estilos"/>
+					<Item id="2227" name="Forzar a cor de fondo para tódolos estilos"/>
+					<Item id="2228" name="Forzar a elección do tipo de letra para todos los estilos"/>
+					<Item id="2229" name="Forzar tamaño de fonte para tódolos estilos"/>
+					<Item id="2230" name="Forzar grosa para tódolos estilos"/>
+					<Item id="2231" name="Forzar cursiva para tódolos estilos"/>
+					<Item id="2232" name="Forzar subliñado para tódolos estilos"/>
+					<Item id="2234" name="Ir a Configuración"/>
+					<Item id="2235" name="¿Que é a anulación global?"/>
```

Comment:
Don't translate "&quot;Global override&quot;

Reply:
Please check line 1822: `<!-- Don't translate "&quot;Global override&quot; and &quot;Default Style&quot; -->`

Don't translate **" ??** ...